lonestar/analytics/cpu/betweennesscentrality/betweenness_centrality_cli.cpp

```diff
@@ -68,6 +68,12 @@ static cll::opt<BetweennessCentralityPlan::Algorithm> algo(
         ),
     cll::init(BetweennessCentralityPlan::kLevel));
 
+static cll::opt<bool> thread_spin(
+    "threadSpin",
+    cll::desc("If enabled, threads busy-wait for work rather than use "
+              "condition variable (default false)"),
```

Comment:
This seems like a library level option rather than an app specific option

Reply:
that's fair; might be better to move it to a common CLI in the lonestar init later